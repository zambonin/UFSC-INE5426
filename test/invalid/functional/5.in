# second argument on map receives int, expects a int array
int t
int output[10]
output= map(lambda int x -> x + 2, t)
