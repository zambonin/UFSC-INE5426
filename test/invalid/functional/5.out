[Line 4] semantic error: second parameter must be of array type
[Line 6] semantic error: length operation expects an array
[Line 7] semantic error: left hand side of index operation is not an array
[Line 7] semantic error: parameter x expected integer but received undefined
[Line 10] semantic error: attribution operation expected integer array but received integer
